src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -427,9 +428,28 @@ VtolAttitudeControl::is_fixed_wing_requested()
 		to_fw = _transition_command == vehicle_status_s::VEHICLE_VTOL_STATE_FW;
 	}
 
+	if(_abort_transition) {
+		to_fw = false;
+	}
+
 	return to_fw;
 }
 
+/*
+ * Abort transition
+ */
+void
+VtolAttitudeControl::abort_transition()
+{
+	if(!_abort_transition) {
+		// TODO: add mavlink log message
+		_abort_transition = true;
```

Comment:
It should be reset, you are right. I think however that the entire mission (if applicable) should be cancelled as it might not be able to finish.

The naming is based on possible future extention to canceling mc transition, or do you think that is too far fetched?